src/DataViews/include/DataViews/LiveFunctionsDataView.h

```diff
@@ -14,6 +14,7 @@
 #include <utility>
 #include <vector>
 
+#include "../CompareAscendingOrDescending.h"
```

Comment:
Fix this `#include`.

Reply:
What do you mean "fix" ?

Reply:
Don't use `../`

Reply:
Sorry, I don't get it. `CompareAscendingOrDescending.h` is in `src/DataViews`, while `LiveFunctionsDataView.h` is in `src/DataViews/include/DataViews`. How should I include it?

Reply:
That tells you that something is wrong. You shouldn't include module-private headers from module-public headers. This means that `CompareAscendingOrDescending.h` needs to be moved to the `include` directory.

In addition, as `CompareAscendingOrDescending.h` was a private header and its content was supposed to be used only from inside the module, its content should be moved to the `orbit_data_views_internal` namespace.

Of course an alternative would be to not use that header at all from `LiveFunctionsDataView.h`, but you need it in a template so I don't know if that's possible.